### ISSUE TITLE
Out of Memory crash with Radar Chart

### ISSUE THREAD

**Issue by leidig54** [imaNNeoFighT/fl_chart#694]
**Describe the bug**
App hangs trying to build the chart. Eventually an error is thrown (takes 5-10 mins to throw).

```
════════ Exception caught by rendering library ═════════════════════════════════
The following OutOfMemoryError was thrown during paint():
Out of Memory

The relevant error-causing widget was
RadarChart
lib/…/presentation/my_t_home.dart:671
When the exception was thrown, this was the stack
#0      List._growToNextCapacity (dart:core-patch/growable_array.dart:383:5)
#1      RadarChartPainter._drawTicks
package:fl_chart/…/radar_chart/radar_chart_painter.dart:93
#2      RadarChartPainter.paint
package:fl_chart/…/radar_chart/radar_chart_painter.dart:58
#3      RenderRadarChart.paint
package:fl_chart/…/radar_chart/radar_chart_renderer.dart:95
#4      RenderObject._paintWithContext
package:flutter/…/rendering/object.dart:2317
...
The following RenderObject was being processed when the exception was fired: RenderRadarChart#137a0 relayoutBoundary=up16
RenderObject: RenderRadarChart#137a0 relayoutBoundary=up16
    parentData: <none> (can use size)
    constraints: BoxConstraints(0.0<=w<=376.0, h=258.7)
    size: Size(376.0, 258.7)
════════════════════════════════════════════════════════════════════════════════
Exhausted heap space, trying to allocate 2147483664 bytes.

```
**To Reproduce**

```
import 'package:fl_chart/fl_chart.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Expanded(
              child: RadarChart(RadarChartData(dataSets: [
                RadarDataSet(
                  fillColor: Colors.black,
                  dataEntries: [
                    RadarEntry(value: 5),
                  ],
                )
              ])),
            )
          ],
        ),
      ),
    );
  }
}

```

Run on iPhone 11 Simulator. M1 MacBook Air.

**Versions**
```
 [✓] Flutter (Channel stable, 2.2.1, on macOS 11.4 20F71 darwin-arm, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.56.2)
[✓] Connected device (3 available)

• No issues found!
 
  fl_chart: ^0.36.1
```

**Comment by sisierra** [comment]
Hi I had the same issue when I used your code snipped. But as soon as I add a minimum quantity of three RadarEntries the RadarChart can be painted.

So I think you should check if at least three RadarEntries might help in your case.

**Comment by imaNNeoFighT** [comment]
Thanks for reproducible code!

**Comment by imaNNeoFighT** [comment]
Fixed in [0.36.3](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.36.3).
Please check it out.

Thanks for reporting!
